code/online_challenges/src/codechef/MARBLES/MARBLES.py

```diff
@@ -0,0 +1,24 @@
+T = int(raw_input());
+
+while(T>0):
```

Comment:
Reformat using black; port to Python 3; don't use unnecessary parentheses. All variables should be lowercase.